src/wordpress-playground/index.tsx

```diff
@@ -297,3 +299,104 @@ function zipNameToHumanName(zipName) {
 }
 
 main();
+
+async function overwriteFile() {
+	const targetFile = document.getElementById('target-file').value;
+	const targetContent = document.getElementById('target-content').value;
+	console.log(targetFile);
+	console.log(targetContent);
+	await workerThread.writeFile(targetFile, targetContent);
+}
+
+async function generateZip() {
+	await workerThread.run({
+		code: `
+			<?php
```

Comment:
Nit: Any whitespace before `<?php` will make it to PHP's stdout
```suggestion
		code: `<?php
```